scripts/config_R_cuda.sh

```diff
@@ -32,6 +32,8 @@ NVBLAS_GPU_LIST ALL' > /etc/nvblas.conf
 
 echo "NVBLAS_CONFIG_FILE=$NVBLAS_CONFIG_FILE" >> ${R_HOME}/etc/Renviron.site
 
+test_config_cuda.sh /var/log/nvblas.log
```

Comment:
It seems that this line caused the latest build to fail.
Isn't it `/rocker_scripts/test_config_cuda.sh /var/log/nvblas.log` correctly?

Reply:
Oh, right, I was thinking it didn't need the parent directory because they're at the same level.  Sorry about that.
Looks like the directory is called `scripts`.
I've changed as you suggest to:
`/scripts/test_config_cuda.sh /var/log/nvblas.log`